frontend/src/features/verifiable-fields/VerifiableFieldProvider.tsx

```diff
@@ -178,7 +178,7 @@ export const VerifiableFieldProvider = ({
         },
       })
     },
-    [getValues, schema._id, setFocus, setValue, verifyOtpMutation],
+    [getValues, schema._id, setFocus, setValue, verifyFormOtpMutation],
```

Comment:
(assuming adding the word `Form` is to distinguish from login OTPs) maybe this can be done in a separate PR? we can decide on the language beforehand though, no issue.

Reply:
Nah, https://github.com/opengovsg/FormSG/pull/6003#discussion_r1162222358. Will remove `Form`.